.github/workflows/conformance-ginkgo.yaml

```
@@ -39,16 +66,20 @@ concurrency:
   # - Event type
   # - A unique identifier depending on event type:
   #   - schedule: SHA
-  #   - workflow_dispatch: PR number
+  #   - workflow_dispatch: PR number.
+  # The 'ginkgo' suffix added to both schedule and workflow_dispatch ensures
+  # that for workflow_call events it doesn't conflict with other potential
+  # parallel workflow_call done to other workflows from the same parent
+  # workflow. As otherwise the group name would be identical.
   #
   # This structure ensures a unique concurrency group name is generated for each
   # type of testing, such that re-runs will cancel the previous run.
   group: |
     ${{ github.workflow }}
```

I would suggest just putting the hardcoded string here instead, it's far more natural than doing string manipulation on the sha / PR number.
```suggestion
    ${{ github.workflow }}
    ginkgo
```

---

(or if you like, combine it with the line 78 here)